schedule/qam/common/mau-webserver.yaml

```diff
@@ -53,7 +53,6 @@ conditional_schedule:
         - console/django
         - console/nginx
       15-SP5:
-        - console/django
```

Comment:
does this mean it is unscheduled from mau-webserver?

Reply:
I think it's fine to do this like it is for now, but if @foursixnine agrees we could do that

Reply:
I think it's fine to do this like it is for now, but if @foursixnine agrees we could do that

Reply:
@BillAnastasiadis, @rfan1 I trust the team here :), on the bright side is a single schedule to mantain if we also use it for TW and Leap